requirements/env_common.yml

```diff
@@ -24,5 +24,5 @@ dependencies:
     - simdna
```

Comment:
Not related to this PR, but we should remove `simdna` as well when we get a chance. We similarly only use one function from there.